src/blueberry-muffins.md

```diff
@@ -0,0 +1,54 @@
+# Blueberry Muffins
+
+![blueberry-muffins](/data/pix/blueberry-muffins.webp)
+
+- ⏲️ Prep time: 20 min
+- 🍳 Baking time: 20 min
+- 🍽️ Servings: 12
+
+## Equipment
+
+-At least 12 muffin tins / A cake tin and optionally a hand mixer
+
+## Ingredients
+
+- 250g / 9 oz Blueberries 
+- 125g / 4½ oz Butter*
+- 150g / 5½ oz Sugar (Of course  you can lower the amount to your liking)
+- 1 Vanilla Pod or Teaspoon of vanilla sugar
+- 2 Eggs
+- 125ml / 4.22 fluid ounces of Milk
+- 300g /  11 oz Flour
+- 2 Teaspoons of Backing Powder
+- 125g / 4½ oz Powdered sugar (optional)
+- Half a lemon (optional)
+*If you don't have Butter, you can substitute it with 100g of Oil / 3½ oz of Oil suited for baking. Notice the measurement in grams / ounces and not milliliter and fluid ounces
```

Comment:
```suggestion
- Half a lemon (optional)

*If you don't have Butter, you can substitute it with 100g of Oil / 3½ oz of Oil suited for baking. Notice the measurement in grams / ounces and not milliliter and fluid ounces
```

Adding an extra space. Otherwise markdown will consider the `*` line t be appended to the `half a lemon` line